README.md

```diff
@@ -1,8 +1,8 @@
 # react-redux-material_ui-boilerplate
-A boilerplate for React + Redux + Material UI + ES6 syntax applications. This boilerplate includes following tools and frameworks.
+A boilerplate for React + Redux + Material UI + ES6 syntax applications. This boilerplate includes the following tools and frameworks:
 
 * [React](https://facebook.github.io/react/)
-* [Redux](http://rackt.org/redux/index.html)
+* [Redux](https://reduxframework.com/)
```

Comment:
@nicolas-raoul  please modify url to the correct one, then I will merge this PR.